tools/update_files_from_cli.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import pathlib
+import re
+import shlex
+import subprocess
+import sys
+import typing
+
+import yaml
+
+
+def print_to_err_and_exit(msg) -> None:
+    print(msg, file=sys.stderr)
+    sys.exit(1)
+
+
+def file_type(filepath: str) -> str:
+    path = os.path.expanduser(filepath)
+    if not os.path.isfile(path):
+        raise argparse.ArgumentTypeError(
+            f"The specified file path '{filepath}' is invalid"
+        )
+    return path
+
+
+def get_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(description="UpdateCLI")
+
+    parser.add_argument(
+        "-c",
+        "--config",
+        action="store",
+        type=file_type,
```

Comment:
Not really because `argparse.FileType` requires a mode and directly open that file with that mode, but in the script i need to open it at different times with different modes.